src/emulate.c

```diff
@@ -1336,12 +1354,14 @@ void rv_step(riscv_t *rv, int32_t cycles)
 void ebreak_handler(riscv_t *rv)
 {
     assert(rv);
+    rv->compressed = false;
     rv_except_breakpoint(rv, rv->PC);
 }
 
 void ecall_handler(riscv_t *rv)
 {
     assert(rv);
+    rv->compressed = false;
```

Comment:
For the consistency, I think we can move it to the function `emulate`.

Reply:
OK